test/ctia/entity/web_test.clj

```diff
@@ -261,12 +260,13 @@
      :bad-iss-jwt-2 (-> claims-2 (assoc :iss "IROH Auth") jwt/jwt (jwt/sign :RS256 priv-key-2) jwt/to-str)}))
 
 (s/defn apply-fixtures-with-app
-  [properties f-with-app :-  (s/=> s/Any (s/=> s/Any (s/named s/Any 'app)))]
+  [properties f-with-app :- (s/=> s/Any (s/=> s/Any (s/named s/Any 'app)))]
   (let [fixture-fn
         (join-fixtures [helpers/fixture-log
-                        helpers/fixture-properties:clean
                         helpers/fixture-properties:cors
-                        #(helpers/with-properties properties (%))
+                        #(helpers/with-properties
+                           (conj properties "ctia.auth.type" "allow-all")
```

Comment:
There were 2 overlapping calls to `helpers/fixture-properties:clean` in this ns that this PR removes. AFAICT `ctia.auth.type` is the only important property for the latter call, so a straightforward fix is to just set it explicitly.